modules/unsupported/mbstyle/src/main/java/org/geotools/mbstyle/parse/MBObjectParser.java

```diff
@@ -885,4 +1113,65 @@ public Expression bool(JSONObject json, String tag, boolean fallback)
                     + obj.getClass().getSimpleName());
         }
     }
+    
+
+    /**
+     * Maps a json value at 'tag' in the provided JSONObject to a {@link Displacement}.
+     * 
+     * @param json The JSONObject in which to look up a displacement value
+     * @param tag The tag in the JSONObject
+     * @param fallback The fallback displacement, if no value is found at that tag.
+     * @return A displacement from the json
+     */
+    public Displacement displacement(JSONObject json, String tag, Displacement fallback) {
```

Comment:
I do not think displacement is working (see geotools bug ticket). Let's keep this ticket and fix the geotools renderer.